### PR TITLE
Dashboard tests corral package

### PR DESCRIPTION
**Pull request by izaac**
This package is for use of Dashboard UI tests automation.

It created a single node from the Jenkins executor that meets the requirements for the tests execution.
Then executes the logic in the dashboard-tests package.

### CODE REVIEW

File: templates/aws/single_node/terraform/pools/main.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+terraform {
+  required_version = ">= 0.13.1"
+  required_providers {
```
Comment (igomez06) [inline templates/aws/single_node/terraform/pools/main.tf:3]: Is there a reason you're not using this template for a single node? https://github.com/rancherlabs/corral-packages/blob/main/templates/aws/nodes/manifest.yaml 
This template can be used to create a single. Let's prevent creating terraform packages that we don't need.

**Review by igomez06 — COMMENTED** [review]
just one comment about the terraform

File: templates/aws/single_node/terraform/pools/main.tf (hunk 1 above)
Comment (izaac) [reply templates/aws/single_node/terraform/pools/main.tf]: updated

**Review by igomez06 — APPROVED** [review]
LGTM

**Comment by izaac** [comment]
I've mentioned that I was having errors on this change, but it was just a missing variable.

`bastion_ip` appears as optional but terraform was requiring it during corral create. Setting it as empty string works.

**Comment by izaac** [comment]
This will code will be replaced with the latest recurring test feature.

**Pull request closed without merging**